src/i3pager.h

```diff
@@ -33,5 +41,4 @@ public Q_SLOTS:
     void handleI3Events();
 };
 
-
-#endif //I3PAGER_PLASMOID_H
+#endif I3PAGER_PLASMOID_H
```

Comment:
When compiling I'm getting some warnings about these endif changes:
```
i3-pager/src/i3pager.h:44:8: warning: extra tokens at end of #endif directive [-Wendif-labels]
   44 | #endif I3PAGER_PLASMOID_H
```
I don't know C++ that well, but from some quick Googling it seems like endif with a comment is more correct?

Reply:
I don't know C++ that well either.. You're absolutely right, what I wrote doesn't make any sense. I'll fix it.